### DISCUSSION TITLE
Query related to v9

### DISCUSSION THREAD

**Discussion by amitdev-code** [directus/directus#4440] [Archive]
Dear Team,
I am just confused like if i start using the initial release of v9 what consequences i may face in next upgrade release of directus v9, is there possibility that upgrade may crash my application.because of this all reason i was thinking to go with v8 as there might be no major changes, will this be a right choice??

**Comment by rijkvanzanten** [comment]
While there might be small tweaks and changes that otherwise would be considered breaking changes between v9 RC releases, I would not suggest starting a project on v8 this close to the final v9 release. v9 is generally speaking more stable, feature rich, and performant, and v8 won't be supported moving forwards.

See https://github.com/directus/directus/milestones for everything that's left to do before we're ready to release it as v9.0.0

**Reply by benhaynes** [reply]
⚡

**Comment by benhaynes** [comment]
We are already writing migrations for all v9-RC releases, so I don't think there's a big risk of any big changes. Going with v8 would be a mistake at this point (unless you _require_ PHP), since that codebase will not be officially maintained moving forward and the migration process from v8 to v9 is a pain due to breaking changes between the two.
